tests/submit_tests/test_fund.py

```diff
@@ -45,66 +46,3 @@ def test_get_active_funds_raises_value_error(test_fund_service):
     fund_configs = test_fund_service.get_active_funds(["Non-existent Role"])
 
     assert not fund_configs
-
-
-def test_fund_config_validations():
-    valid_fund_attrs = dict(
-        fund_name="Test Fund Name",
-        current_reporting_period="Test Reporting Period",
-        current_reporting_round=1,
-        current_deadline=datetime.date(day=1, month=1, year=2001),
-        email="testemail@test.gov.uk",
-        active=True,
-        auth_class=AuthBase,
-        user_role="Test Role",
-        fund_code="TF",
-    )
-
-    # success
-    FundConfig(**valid_fund_attrs)
-
-    # fails
-    with pytest.raises(AssertionError):
-        invalid_attrs = copy(valid_fund_attrs)
-        invalid_attrs["fund_name"] = 1234  # not a string
-        FundConfig(**invalid_attrs)
-
-    with pytest.raises(AssertionError):
-        invalid_attrs = copy(valid_fund_attrs)
-        invalid_attrs["current_reporting_period"] = 1234  # not a string
-        FundConfig(**invalid_attrs)
-
-    with pytest.raises(AssertionError):
-        invalid_attrs = copy(valid_fund_attrs)
-        invalid_attrs["current_reporting_round"] = "1234"  # not an int
-        FundConfig(**invalid_attrs)
-
-    with pytest.raises(AssertionError):
-        invalid_attrs = copy(valid_fund_attrs)
-        invalid_attrs["current_deadline"] = "1/1/2001"  # not a datetime
-        FundConfig(**invalid_attrs)
-
-    with pytest.raises(AssertionError):
-        invalid_attrs = copy(valid_fund_attrs)
-        invalid_attrs["email"] = 1234  # not a string
```

Comment:
Yes - because that validation was removed? Feel free to review commit by commit, I've chunked out to explain this

Reply:
I think we wouldn't want to have a different email for each fund if we were planning on onboarding any more funds to Submit, but since we're not I think it will be quicker just to let TF have the unique content they've asked for (as long as the tech side of having a separate email is quick to implement). Let's go with the unique content for TF if that's ok.